doc/ref_model/chapters/chapter05.md

```diff
@@ -214,50 +236,18 @@ This section will detail Cloud Infrastructure Software Profiles and associated c
 
 The support of a variety of different workload types, each with different (sometimes conflicting) compute, storage, and network characteristics, including accelerations and optimizations, drives the need to aggregate these characteristics as a hardware (host) profile and capabilities. A host profile is essentially a “personality” assigned to a compute host (physical server, also known as compute host, host, node, or pServer). The host profiles and related capabilities consist of the intrinsic compute host capabilities (such as number of CPU sockets, number of cores per CPU, RAM, local disks and their capacity, etc.), and capabilities enabled in hardware/BIOS, specialised hardware (such as accelerators), the underlay networking, and storage.
 
-This chapter defines a simplified host, host profile and related capabilities model associated with each of the different Cloud Infrastructure Hardware Profile and related capabilities; some of these profiles and capability parameters are shown in **Figure 5-4**.
+This chapter defines a simplified host, profile and related capabilities model associated with each of the different Cloud Infrastructure Hardware Profile and related capabilities; the [two profiles](./chapter02.md#241-node-profiles-top-level-partitions) (aka host profiles, node profiles, hardware profiles) and some of their associated capabilities are shown in **Figure 5-4**.
 
-<p align="center"><img src="../figures/RM_chap5_fig_5_4_HW_profile.png" alt="ref_hw_profiles" title="Reference HW Profiles" width="100%"/></p>
+<p align="center"><img src="../figures/RM-ch05-hw-profile.png" alt="ref_hw_profiles" title="Reference HW Profiles" width="80%"/></p>
```

Comment:
Doesn't show Figure 5-4 before changing URL.

Reply:
@petorre Can you check again. It shows up for me.

Reply:
@pgoyal01 : Yes, it does - I see it when I go to your patch. Somehow in PR "File changes" "Display rich diff" it doesn't show for me. Sorry.